README.md

```diff
@@ -1,4 +1,285 @@
-# shapeup (WIP)
+# shapeup
 
-This library provides the shape and reshape React property types, including
-the ability to declare frozen shapes.
+The React `shape` [property type](https://www.npmjs.com/package/prop-types) is
+useful for declaring how objects provided to components should look like. By
+stating that a property has a given shape, we are establishing a contract that
+must be fulfilled when instantiating the component. However, prop-types' shapes
+are to be considered more as a minimum requirement than as a proper external
+interface declaration: providing an object that has a superset of the declared
+properties is not considered an error, and therefore components can end up
+relying on fields that are not part of the contract.
+
+The shapeup library fixes this by requiring the provided properties to only
+have the declared set of fields: when using shapeup, passing in an object with
```

Comment:
The shapeup library fixes this by forcing the provided properties to only be from the declared set of fields

Reply:
Why the : in the sentence? Should they just be periods. It's here and in the previous paragraph as well.

Reply:
Done.